pandas/api/tests/test_api.py

```diff
@@ -229,6 +229,6 @@ def test_deprecation_access_obj(self):
             pd.datetools.monthEnd
 
 if __name__ == '__main__':
-    import nose
-    nose.runmodule(argv=[__file__, '-vvs', '-x', '--pdb', '--pdb-failure'],
-                   exit=False)
+    import pytest
+
+    pytest.main([__file__, '-vvs', '-x', '--pdb'])
```

Comment:
yeah I agree. I think we should do a separate PR to remove ALL of these (makes this PR a bit simpler too).

we don't advocate actually running indivdidual files, rather it is appropriate to use the test framework.